katsdpcontroller/generator.py

```diff
@@ -347,6 +347,53 @@ def _make_fgpu(
     g.add_node(dst_multicast)
     g.add_edge(dst_multicast, fgpu_group, depends_init=True, depends_ready=True)
 
+    static_sensors = [
+        Sensor(float, f"{stream.name}-adc-sample-rate",
+               "Sample rate of the ADC",
+               "Hz",
+               default=stream.adc_sample_rate, initial_status=Sensor.Status.NOMINAL),
+        Sensor(float, f"{stream.name}-bandwidth",
+               "The analogue bandwidth of the digitised band",
+               "Hz",
+               default=stream.bandwidth, initial_status=Sensor.Status.NOMINAL),
+        # The timestamps are simply ADC sample counts
+        Sensor(float, f"{stream.name}-scale-factor-timestamp",
+               "Factor by which to divide instrument timestamps to convert to unix seconds",
+               "Hz",
+               default=stream.adc_sample_rate, initial_status=Sensor.Status.NOMINAL),
+        Sensor(float, f"{stream.name}-sync-time",
+               "The time at which the digitisers were synchronised. Seconds since the Unix Epoch.",
+               "s",
+               default=float(sync_time), initial_status=Sensor.Status.NOMINAL),
+        Sensor(int, f"{stream.name}-n-ants",
+               "The number of antennas in the instrument",
+               default=len(stream.antennas), initial_status=Sensor.Status.NOMINAL),
+        Sensor(int, f"{stream.name}-n-inputs",
+               "The number of single polarisation inputs to the instrument",
```

Comment:
```suggestion
               "The number of single-polarisation inputs to the instrument",
```
I'm not sure what the ICD says but I'm fairly sure that grammatically there should be a hyphen here.